install_docker.sh

```diff
@@ -0,0 +1,8 @@
+sudo apt-get update
```

Comment:
Can you please add a comment here about how this is used? I'm guessing this is run on the baremetal host and `add_partition.sh` runs inside the docker container?

Reply:
Yes, `add_partition.sh` and `install_docker.sh` should be used together. `add_partition.sh` is for extending the space of / path in cloudlab machines. Then run `install_docker.sh` to install docker and let the docker to write all its images and other stuff to this new parttion.